index.html

```diff
@@ -65,7 +65,21 @@
           w3cid: 41758
         }],
 
-        github: "https://github.com/w3c/vc-di-eddsa/",
+        // extend the bibliography entries
+        //localBiblio: webpayments.localBiblio,
+
+        wg: "Credentials Community Group",
+        // URI of the public WG page
+        wgURI: "https://www.w3.org/community/credentials/",
+        // name (with the @w3c.org) of the public mailing to which comments are due
+        wgPublicList: "public-credentials",
+        // URI of the patent status for this WG, for Rec-track documents
+        // !!!! IMPORTANT !!!!
+        // This is important for Rec-track documents, do not copy a patent URI from a random
+        // document unless you know what you're doing. If in doubt ask your friendly neighbourhood
+        // Team Contact.
+        wgPatentURI: "https://www.w3.org/community/about/agreements/cla/",
+        github: "https://github.com/w3c-ccg/di-eddsa-2020/",
```

Comment:
Looks like this change should be reverted.

Reply:
Any easy way to do that?

Reply:
Changes should be reverted. Started with w3c main then added back in test vectors, and hand crafting indentation.